Whiskey/Whiskey.psd1

```diff
@@ -149,6 +149,7 @@
             # ReleaseNotes of this module
             ReleaseNotes = @'
 * Fixed: PowerShell task fails when Whiskey is stored in a directory that isn't named "Whiskey".
+* Fixed: NuGetPush task fails when pushing to nuget.org.

```

Comment:
Can you point out how/where the NuGetPush task was fixed in regards to pushing to nuget.org? All I see is a property was added to allow skipping the check for the package after it's been uploaded.

Reply:
@jlariono Fixed. Ready for re-review.